main/SQLHelper.cpp

```diff
@@ -4017,8 +4035,11 @@ void CSQLHelper::UpdateRainLog()
 			if (splitresults.size()<2)
 				continue; //impossible
 
-			int rate=atoi(splitresults[0].c_str());
-			float total = static_cast<float>(atof(splitresults[1].c_str()));
+			float total=0;
+			int rate=0;
+
+			rate=atoi(splitresults[0].c_str());
+			total = static_cast<float>(atof(splitresults[1].c_str()));
```

Comment:
Sorry, I uploaded my version of SQLHelper.cpp and it had this code, so I reverted some changes from my versión (5231). I'll revert all code I did'nt modified. Youre right this patch is just for energy/power.
Sorry it's my first pull.